kodilla-stream/src/test/java/com/kodilla/exception/homework/WarehouseTestSuite.java

```diff
@@ -0,0 +1,31 @@
+package com.kodilla.exception.homework;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class WarehouseTestSuite {
+
+    private Warehouse warehouse = new Warehouse();
+
+    @Test
+    public void testGetOrderExists() throws OrderDoesntExistException {
+        // given
+        warehouse.addOrder(new Order("5/5/2020"));
+
+        // when
+        Order result = warehouse.getOrder("5/5/2020");
+
+        // then
+        assertEquals("5/5/2020", result.getNumber());
+    }
+
+    @Test(expected = OrderDoesntExistException.class)
+    public void testGetOrderDoseNotExists() throws OrderDoesntExistException {
+        // when
+        Order result = warehouse.getOrder("5/05/2020");
+
+        // then
```

Comment:
a czy to sie zdarzy? W ogole ci ten test przeszedl? Bo na moje oko on nie przechodzi. W linii 26 powinien byc taki numer zamowienia, aby zamowienie nie istnialo.

Reply:
tak. mi ten test przeszedł: